.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Integation tests
+"on":
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  maven-it:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java: [8, 11, 13]
```

Comment:
@olenagerasimova minimum Java version for Artipie is 11 at the moment, please remove Java 8 from versions list